src/main/webapp/app/assessment-shared/assessment-warning/assessment-warning.component.html

```diff
@@ -0,0 +1,8 @@
+<fa-icon
```

Comment:
Just personal preference: you can add the html directly in the `assessment-warning.component.ts` since it's just 8 lines of code.